vendor/@carbon/react.js

```diff
@@ -0,0 +1,8 @@
+const { returnOrThrow } = require('../../helper.js');
+
+/**
+ * Use this to access Carbon icons globally in UI extensions.
+ *
+ * @type {import('@carbon/react')}
+ */
+module.exports = returnOrThrow(() => window.vendor?.carbonReact, '5.33');
```

Comment:
```suggestion
module.exports = returnOrThrow(() => window.vendor?.carbonReact, '5.38');
```

It is only going to be available in the future version.